main.go

```diff
@@ -53,6 +56,8 @@ func getFlags() (flags CommandFlags) {
 	flag.StringVar(&flags.jsonAPIAddr, "j", ":7001", "JSON API Address")
 	flag.StringVar(&flags.certFile, "cert", "", "HTTPS Cert file of JSON API")
 	flag.StringVar(&flags.keyFile, "key", "", "HTTPS Key file of JSON API")
+	flag.StringVar(&flags.apiUser, "u", "", "User for authentication of JSON API")
+	flag.StringVar(&flags.apiPassword, "p", "", "Password for authentication of JSON API")
 	flag.Parse()
 	return
```

Comment:
`return` cannot be omitted here because every go function with return value(`flag`) must contain a `return` statement